src/components/pages/StudyCase/EditStudyCasePage.jsx

```diff
@@ -73,11 +70,12 @@ class EditStudyCasePage extends React.Component {
         project_bmes_attributes: nextProps.studyCases.included
           .filter(sc => sc.type === 'project_bmes')
           .filter(pBme => !!pBme.relationships.bme.data)
-          .map(pBme => ({
+          .map((pBme, index) => ({
             id: pBme.id,
-            bme_id: pBme.relationships.bme.data.id,
+            category_id: pBme.relationships.bme.data.id,
```

Comment:
Names don't add up. Why are we storing a BME's id on something named `category_id`?